tests/fixtures/misc/checker/log1p.py

```diff
@@ -0,0 +1,6 @@
+import torch
+a = torch.randn(5)
+b = torch.log(1 + a)
+c = torch.log(a + 1)
+b = torch.log(1.0 + a)
+c = torch.log(a + 1.0)
```

Comment:
Could we already add Array API examples here? We don't have the capability of automatically detecting them at the moment, but it's good to be explicit about the known false negatives, especially when we can catch them later

```python
import torch

a = torch.randn(5)

print((a + 1).log())
print(a.log1p())
```

Reply:
Added a false negative.